crates/astria-conductor/src/celestia/mod.rs

```diff
@@ -112,13 +116,19 @@ impl GetSequencerHeight for ReconstructedBlock {
     }
 }
 
+pub(super) struct ReconstructedBlocks {
+    celestia_height: u64,
+    sequencer_namespace: Namespace,
+    rollup_namespace: Namespace,
```

Comment:
shouldn't these always be known since they're set on startup?

Reply:
Yes, they are - I used them for reporting, but that seems redundant now (because the namespaces are always part of the `FetchConvertAndVerify::execute` instrumented future. Removed.